client/src/App.js

```diff
@@ -22,7 +23,11 @@ export default class App extends Component {
     return (
       <div>
         <SavedList list={this.state.savedList} />
-        <div>Replace this Div with your Routes</div>
+        <Route exact path="/" component={MovieList} />
+        <Route exact path="/movies/:id" 
```

Comment:
you wont need the exact prop on the '/movies/:id' path in this instance, just on the '/' path